cfgov/jinja2/v1/_includes/organisms/job-listing-list.html

```diff
@@ -30,10 +30,10 @@ <h3>
         <li class="m-list_item">
             <a class="m-list_link"
                href="{{ get_protected_url(job) }}">{{ job.title }}</a>
-            <p class="a-date">
```

Comment:
Hmm, so we now only use a-date in related posts. I wonder if it should stay in Capital Framework.

Reply:
@anselmbradford That's a good question -- if it's no longer our only date display option...